tests/datatypes/test_bytes.py

```diff
@@ -349,6 +349,26 @@ def test_upper(self):
             print(b'\x46\x55\x43\x4B'.upper())
         """)
 
+    def test_ljust(self):
+        self.assertCodeExecution("""
+            print(b'testMoreThanWidth'.ljust(5))
+            print(b'testEqualWidth'.ljust(14))
+            print(b'testLessThanWidth'.ljust(20))
+            print(b'testMoreWithFill'.ljust(2, b'x'))
+            print(b'testEqualWithFill'.ljust(17, b'x'))
+            print(b'testLessWithFill'.ljust(25, b'x'))
+            print(b'testNegative'.ljust(-20))
+            print(b''.ljust(5))
+            print(b'testNoChangeWidthOne'.ljust(True, b'x'))
+            print(b'testBArraySecondArg'.ljust(True, bytearray(b'x')))
+            try:
+                print(b'testStrArgError'.ljust('5'))
+                print(b'testMoreLengthError'.ljust(12, b'as'))
+                print(b'testStrFillingChar'.ljust(12, 'a'))
```

Comment:
You need to wrap each individual test in a try-catch; otherwise, the first test will raise an exception, and the second two wont ever be executed.